build.gradle

```diff
@@ -27,6 +27,8 @@ repositories {
 		name = "Fabric"
 		url = 'https://maven.fabricmc.net/'
 	}
+
+	mavenLocal()
```

Comment:
Should be removed now?

Reply:
It has been merged and published to the Fabric maven now.

Reply:
yeah I just did that